tests/framework/utils_ftrace.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
was it created in 2020?

Reply:
ugh, every damn time I add a file.

I wonder if it'd be possible to add a test for this. E.g. "if file is new, assert copyright is current year". But I guess that would break if we move files around, or add some external file that actually needs to have an older copyright.

anyway, fixed :)